Resources/AzureDevOps/azure-pipelines.yml

```diff
@@ -0,0 +1,60 @@
+variables:
+  SolutionBaseName: 'NCS.DSS.ContentPushService'
+  BuildPlatform: 'any cpu'
+  BuildConfiguration: 'release'
+
+pool: 
+  name: 'NCS - CI and CD'
+  demands:
+    - msbuild
+    - visualstudio
+
+resources:
+  repositories:
+  - repository: self
+  - repository: dfc-devops
+    type: github
+    name: SkillsFundingAgency/dfc-devops
+    ref: refs/tags/v1.10.4
+    endpoint: 'GitHub (CDH)'
+
+jobs:
+# Test the branch name, the output of this job will satisfy the condition of one or neither of the subsequent jobs
+- job: TestBranchName
```

Comment:
do we need this anymore? Previously it controlled a lot of conditional builds.. but the step template from dfc-devops doesn't have this.

Reply:
I'm leaving it in to enforce consistent names, might as well remain consistent within this project.  Not strictly necessary for this pipeline though